src/pyenphase/envoy.py

```diff
@@ -58,9 +59,12 @@ def create_no_verify_ssl_context() -> ssl.SSLContext:
 
 class SupportedFeatures(enum.IntFlag):
     INVERTERS = 1
-    DRY_CONTACTS = 2
-    ENCHARGE = 4
-    ENPOWER = 8
+    METERING = 2
+    TOTAL_CONSUMPTION = 4
+    NET_CONSUMPTION = 8
+    DRY_CONTACTS = 16
```

Comment:
What's the reason dry contacts changed from 2 to 16? Like is there a technical reason for it? I'm curious

Reply:
These could be auto as well. I was moving more likely features lower in the list for historical reasons (when we come look at this 10 years from now). It has no actual impact